src/test/scala/edu/ie3/mobsim/model/ChargingBehaviorSpec.scala

```diff
@@ -368,5 +369,33 @@ class ChargingBehaviorSpec extends UnitSpec with ChargingBehaviorTestData {
         UUID.fromString("7537c0b6-3137-4e30-8a95-db1c0f9d9b81") -> 5.0
       )
     }
+
+    "return the correct price rating" in {
+      val priceRating = PrivateMethod[Double](Symbol("priceRating"))
+
+      val evWithLowPriceMemory: ElectricVehicle = ev1
+      evWithLowPriceMemory.updateChargingPricesMemory(0.0)
+
+      val evWithHighPriceMemory: ElectricVehicle = ev1
+      evWithHighPriceMemory.updateChargingPricesMemory(0.0)
+      evWithHighPriceMemory.updateChargingPricesMemory(1.0)
+
+      val cases = Table(
+        ("chargingStation", "ev", "prices", "resultingRating"),
```

Comment:
I'd rename `resultingRating` to `expectedRating`, since this is the naming we use in tests